cdc/sinkv2/eventsink/txn/mysql/mysql.go

```diff
@@ -99,8 +102,15 @@ func NewMySQLBackends(
 		return nil, err
 	}
 
-	db.SetMaxIdleConns(cfg.WorkerCount)
-	db.SetMaxOpenConns(cfg.WorkerCount)
+	db.SetMaxIdleConns(cfg.WorkerCount + 1)
+	db.SetMaxOpenConns(cfg.WorkerCount + 1)
```

Comment:
Please add a comment about why plus one here

Reply:
got it, will followup